projects/1.21-fabric/assets/combat-roll/combat_roll/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "combat_roll.mod_name":"战斗翻滚",
+
+  "keybinds.combat_roll.roll":"翻滚",
+
+  "gui.combat_roll.config_menu":"战斗翻滚配置",
+  "gui.combat_roll.close":"关闭",
+  "gui.combat_roll.settings":"设置",
+  "gui.combat_roll.hud":"移动闪避箭头",
+  "gui.combat_roll.corner":"换一个角落",
+  "gui.combat_roll.reset":"重置设置",
+
+  "enchantment.combat_roll.longfooted":"长足",
+  "enchantment.combat_roll.longfooted.desc":"增长你的翻滚距离",
+  "enchantment.combat_roll.acrobat":"杂技",
+  "enchantment.combat_roll.acrobat.desc":"减少翻滚之间的冷却时间",
+  "enchantment.combat_roll.multi_roll":"多次翻滚",
```

```suggestion
  "enchantment.combat_roll.multi_roll":"多重翻滚",
```
参见[维基](https://zh.minecraft.wiki/w/Dungeons:%E5%A4%9A%E9%87%8D%E7%BF%BB%E6%BB%9A)